src/sender.js

```diff
@@ -126,6 +126,7 @@ class Sender {
                         await self.close();
                     }
                 });
+            this.socket.setKeepAlive(true);
```

Comment:
nit: we could let the user disabling keep-alive via config, but TBH I don't see a scenario when it's required.